### ISSUE TITLE
Find Out How To Extract Weights From scikit-learn

### ISSUE THREAD

**Issue by andydmh** [andydmh/Machine-Learning-Group9#14]
For comparison purposes we need to train an skit-learn regression model and compare their squared loss error with ours. So we need to figure it out either how to get the loss or how to extract their learned weights to call our J function and calculate the loss. It is also important to understand how their weights are organized (if we can not get the loss directly). In our case we are assuming the bias term is at the beginning that is in w = [w0, w1, w2, ..., wn]  w0 is our bias term (this is important for our loss function).
